GPU/Common/VertexDecoderArm.cpp

```diff
@@ -872,22 +872,14 @@ void VertexDecoderJitCache::Jit_NormalFloat() {
 	STMIA(scratchReg, false, 3, tempReg1, tempReg2, tempReg3);
 }
 
-// Through expands into floats, always. Might want to look at changing this.
 void VertexDecoderJitCache::Jit_PosS8Through() {
```

Comment:
Should this be renamed PosInvalid too?

Reply:
Well, "invalid" I used for when no position is specified at all, and the verts are all culled.

However, when a position is specified, as s8 in through, it is not culled.  It decodes to zero.  So it's kinda invalid, but different from "invalid position in throughmode (no bits set)" which has different behavior.

-[Unknown]

Reply:
As to Arm/Arm64/etc. - I figured might as well keep these jtited, although they're hopefully rarely being hit.  I didn't think it was worth bothering to newly jit invalid, though.

-[Unknown]

Reply:
Ok, cool. Thanks for the clarifications!